bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -122,297 +109,175 @@ If you did not manually create things in the *.things file, the Netatmo Binding
-| DateMinTempThisMonth| DateTime             | Date when minimum temperature was reached this month     |
-| DateMaxTemp         | DateTime             | Date when maximum temperature was reached on current day |
-| DateMaxTempThisWeek | DateTime             | Date when maximum temperature was reached this week      |
-| DateMaxTempThisMonth| DateTime             | Date when maximum temperature was reached this month     |
-| TimeStamp           | DateTime             | Timestamp when data was measured                         |
-| LastMessage         | DateTime             | Last message emitted by the module                       |
-| LowBattery          | Switch               | Low battery                                              |
-| BatteryVP           | Number               | Battery level                                            |
-| RfStatus            | Number               | Signal strength (0 for no signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
+| Channel Group       | Channel Id           | Item Type            | Description                                      |
+|---------------------|----------------------|----------------------|--------------------------------------------------|
+| pressure            | value                | Number:Pressure      | Current pressure                                 |
+| pressure            | absolute             | Number:Pressure      | Pressure at sea level                            |
+| pressure            | trend                | String               | Pressure evolution trend over time               |
+| noise               | value                | Number:Dimensionless | Current noise level                              |
+| humidity            | value                | Number:Dimensionless | Current humidity                                 |
+| humidity            | humidex              | Number               | Computed Humidex index                           |
+| humidity            | humidex-scale        | Number               | Humidex index appreciation                       |
+| humidity            | heat-index           | Number:Temperature   | Computed Heat Index                              |
+| humidity            | dewpoint             | Number:Temperature   | Computed dewpoint temperature                    |
+| humidity            | dew-point-depression | Number:Temperature   | Computed dewpoint depression                     |
+| temperature         | value                | Number:Temperature   | Current temperature                              |
+| temperature         | today-min            | Number:Temperature   | Minimum temperature on current day               |
+| temperature         | today-max            | Number:Temperature   | Maximum temperature on current day               |
+| temperature         | min-today            | Number:Temperature   | Timestamp of today's minimum temperature         |
```

Comment:
Actual channel id seems to be `min-time` and type is DateTime.

Reply:
corrected.